requirements/quality-extra.txt

```diff
@@ -0,0 +1,2 @@
+pylint
+edx-lint                  # edX pylint rules and plugins
```

Comment:
quality.txt is generated, but the output is different depending on when you run it (based on currently available library versions).  The idea is to use the exact same versions consistently until we make a deliberate choice to upgrade them, so we need the output at a particular point in time checked in.